src/therapy/query.py

```diff
@@ -489,7 +483,7 @@ def _get_concept_mapping(
                     mappings = [
                         get_disease_concept_mapping(
                             concept_id=indication.normalized_disease_id,
-                            relation=Relation.RELATED_MATCH,
+                            relation=Relation.EXACT_MATCH,
```

Comment:
Since we're using the `normalized_disease_id` then I believe this should have exact match since we ingest this source